c2corg_ui/static/js/contexthelp.js

```diff
@@ -0,0 +1,76 @@
+goog.provide('app.contextHelpDirective');
+goog.provide('app.ContextHelpModalController');
+
+goog.require('app');
+
+/**
+ * This directive is used to display a contextual help modal dialog.
+ *
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.contextHelpDirective = function($uibModal, gettextCatalog) {
+  return {
+    restrict: 'A',
+    link: function(scope, element, attrs) {
+      element.append('<span class="glyphicon glyphicon-info-sign context-help-sign">');
+      element.on('click', function() {
+        $uibModal.open({
+          controller: 'AppContextHelpModalController',
+          controllerAs: '$ctrl',
```

Comment:
Are you sure ``$ctrl`` is a relevant controller name? It looks like a variable whereas it is just a string.
At least it is not very consistent with other directives whose controller name is something like ``fooCtrl``.